lib/utils/file_download_util.dart

```diff
@@ -0,0 +1,65 @@
+import 'dart:io' as io;
+import 'dart:typed_data';
+import 'package:dio/dio.dart';
+import 'package:flutter_sodium/flutter_sodium.dart';
+import 'package:logging/logging.dart';
+import 'package:photos/core/configuration.dart';
+import 'package:photos/core/network.dart';
+import 'package:photos/models/file.dart' as ente;
+import 'package:photos/services/collections_service.dart';
+
+import 'crypto_util.dart';
+
+final _logger = Logger("file_download_util");
+
+Future<io.File> downloadAndDecrypt(ente.File file,
```

Comment:
I'm skipping reviewing this assuming it's a direct copy-pasta.

Reply:
Ack